### PR TITLE
callback

### PR DESCRIPTION
**Pull request by GurgenBaghdasaryan**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+let persons = ["Aro", "Gugo"];
+
+function htmlMessages(callback) {
```
Comment (edotnet) [inline index.js]: no need for callback here

File: index.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    let item = document.createElement("li");
+    item.innerText = person;
+    document.body.append(item);
+  });
+}
+
+function foo2() {
+  persons.push("asasdas");
+
+  console.log(persons);
```
Comment (edotnet) [inline index.js]: remove log

File: index.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+let persons = ["Aro", "Gugo"];
+
+function htmlMessages(callback) {
+  callback();
+
+  persons.forEach((person) => {
+    let item = document.createElement("li");
+    item.innerText = person;
+    document.body.append(item);
```
Comment (edotnet) [inline index.js]: append `li` items to `ul` element not to `body`

File: callback.js [hunk 4]
```diff
@@ -1,19 +1,24 @@
...
+     `;
+    });
+    addName.innerHTML = htmlMessage;
+  }, 2000);
 }
 
-function foo2() {
-  persons.push("asasdas");
-  console.log(persons);
+function foo2(name, callback) {
```
Comment (arayik-99) [inline callback.js]: Why is the func's name **foo2**

**Pull request merged**